### ISSUE TITLE
Error when typecasting a negative value without a space

### ISSUE THREAD

**Issue by docgloucester** [42School/norminette#197]
When fed the following code :
```C
int	main(void)
{
	int				a;
	unsigned int	b[1];

	a = -10;
	*b = (unsigned int)-a;
	return (0);
}
```

the norminette raises the following error :
`Error: SPC_BFR_OPERATOR     (line:   7, col:  24):      missing space before operator`

**Additional infos**

OS: MacOS Mojave
python --version: Python 3.7.4
norminette -v: norminette 3.3.2

**Comment by docgloucester** [comment]
While the problem can be circumvented by using a space, you'll agree that the fact that typecasting a negative value follows different formatting rules than typecasting a positive one is confusing, especially since the error looks like the minus symbol is misinterpreted for the subtractive operator while it is in this context text the unary operator for negation.

**Comment by N0ich** [comment]
Yes, i will fix that as it not an error on your side, rather on mine.
Thanks for the issue

**Comment by N0ich** [comment]
I dont have the error when running your code with norminette 3.3.2
Can you retry (I've fixed a similar bug in 3.3.2 so it might be it)
